apps/dotcom/client/src/tla/components/dialogs/TlaSignInDialog.tsx

```diff
@@ -209,6 +215,11 @@ function TlaEnterEmailStep({
 							<TlaCtaButton
 								data-testid="tla-google-sign-in-button"
 								className={styles.authCtaButton}
+								onClick={() => {
+									if (!skipRedirect) {
```

Comment:
We'll set up a redirect automatically so that things like file redirects work (this will redirect to the same page as we are currently on). You can opt out of this automatic redirect logic by setting `skipRedirect` - useful when you want to redirect to some other page.